account_invoice_tax_note/tests/test_account_invoice.py

```diff
@@ -48,8 +48,12 @@ def setUp(self):
         taxes = self.tax1 | self.tax2
         self.product1.write({"taxes_id": [(6, False, self.tax1.ids)]})
         self.product2.write({"taxes_id": [(6, False, self.tax2.ids)]})
-        account = self.env["account.account"].search(
-            [("account_type", "=", "income")], limit=1
+        account = self.env["account.account"].create(
+            {
+                "name": "account",
+                "code": "ACCOUNT",
+                "account_type": "income",
+            }
```

Comment:
I think the fix should be:

```suggestion
        account = self.env["account.account"].search(
            [("account_type", "=", "income"), ("company_id", "=", self.env.company.id)], limit=1
        )
```

Reply:
Sure, creating it ensured that it worked in any case (because we might have no localization installed), but your suggestion is less agressive :wink:

Reply:
And avoids other problems: the account creation itself may fail, due to constraint, missing data...

Reply:
@pedrobaeza We have followed your suggestions, tested them, and they work properly with l10n_es. Thanks!